packages/graphback/src/resolvers/knex/KnexResolver.ts

```diff
@@ -0,0 +1,47 @@
+import { BaseKnexResolver } from './BaseKnexResolver'
+
+/**
+ * Resolver implementation for some operations which vary according to database,
+ * extends all the methods of the Base Resolver class
+ * For now only includes the `create` method
+ */
+export class KnexResolver extends BaseKnexResolver {
+  public createTemplate = (database: string, subscription: boolean, fieldName: string, tableName: string, typeName: string): string => {
+    if(database === 'sqlite3' || database === 'mysql') {
```

Comment:
Instead of if's maybe we can have separate classes. 
How much duplication it will be?

Reply:
Just in case we will want to test it later etc. :D

Reply:
Methods or classes ?

Reply:
Classes - we can use strategy pattern and inject different provider based on config. That was initial idea